drivers/acpi/acpi_processor.c

```diff
@@ -880,8 +880,6 @@ int acpi_processor_evaluate_cst(acpi_handle handle, u32 cpu,
 		memcpy(&info->states[++last_index], &cx, sizeof(cx));
 	}
 
-	acpi_handle_info(handle, "Found %d idle states\n", last_index);
-
```

Comment:
can we remove this? I don't know what problem this tries to fix.

Reply:
certainly, this is needed for one of my DUTs, otherwise the dmesg is flooded with this message for some reason and I keep forgetting to drop this when I'm done testing before sending the PR.

I should have a hook to blow up when I try to push this patch to other branch than my wip...